robottelo/host_helpers/contenthost_mixins.py

```diff
@@ -56,6 +56,22 @@ def REPOS(self):
         except KeyError as err:
             raise ValueError(f'Unsupported system version: {self._v_major}') from err
 
+    @cached_property
+    def SATELLITE_CDN_REPOS(self):
+        sat_version = ".".join(settings.server.version.release.split('.')[0:2])
+        return {
+            'satellite': f"satellite-{sat_version}-for-rhel-{self._v_major}-x86_64-rpms",
+            'sat-maintenance': f"satellite-maintenance-{sat_version}-for-rhel-{self._v_major}-x86_64-rpms",
+        }
+
+    @cached_property
+    def CAPSULE_CDN_REPOS(self):
+        sat_version = ".".join(settings.server.version.release.split('.')[0:2])
```

Comment:
`sat_version` could be made a constant to avoid code duplication in this module.

Reply:
@jameerpathan111 Can you be more specific? Where would that constant be placed and where should it take the version data from?

Reply:
In `class VersionedContent`, to reduce code duplication as both `CAPSULE_CDN_REPOS` and `SATELLITE_CDN_REPOS` use it.

Reply:
I will leave this one up to Radek to decide.